modules/asio/1.31.0/MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "asio",
+    version = "1.31.0",
+    compatibility_level = 1,
```

Comment:
could you add `bazel_compatibility = [">=7.2.1"]` for overlay usage?

Reply:
they will pull it in, but just ignore the overlay, causing potentially confusing errors.